### DISCUSSION TITLE
Should we separate stats into another database?

### DISCUSSION THREAD

**Discussion by iyanfdezz** [Arquisoft/wiq_es1a#42] [Announcements]
Should we consider separating user statistics into a separate database? What potential benefits or drawbacks might arise from such a separation? How could this impact system performance and data management?

**Comment by CANCI0** [comment]
Separating user statistics into a separate database could indeed be beneficial in terms of system performance and data management. By doing so, we can potentially reduce the computational cost of calculating data for individual users. Separation could lead to improved system performance. With user statistics in a separate database, the computational load on the main database would be reduced, resulting in faster response times for user queries and transactions. Furthermore, having user statistics in a dedicated database can facilitate more efficient data organization, maintenance, and backup strategies tailored specifically to user-related data.

**Reply by iyanfdezz** [reply]
I think you are right, so I will try to make it work. However, considering that the deadline is next week, I think I'll work on it in a separate branch to incorporate it to  the project later.

**Reply by CANCI0** [reply]
Of course! We can develop it for the next delivery.
